### DISCUSSION TITLE
Beatmap listing audio preview broken on Touch

### DISCUSSION THREAD

**Discussion by nzxl101** [ppy/osu#20218] [Q&A] [answered]
https://user-images.githubusercontent.com/63413558/189474688-1ededba3-e5c4-4c49-9b96-9d9d4dceba82.MOV

It's a bit annoying when I want to download multiple beatmaps, the only way to listen to the preview right now is by clicking on the card.
Normal/Long pressing on ▶️ just takes me to the beatmap page.

Using the latest Testflight (2022.902.1) with iPhone 13 Pro Max @ Full UI Scale

**Comment by peppy** [comment, marked as answer]
Fixed in the next release (https://github.com/ppy/osu/pull/18642)
